auth/configurations/app.ts

```diff
@@ -0,0 +1,6 @@
+let appConfig: any = {};
+
+appConfig.app = process.env.APP || "development";
+appConfig.port = process.env.PORT || 5000;
```

Comment:
If this is a separate application, I think we need this port to be different than the server port.

Reply:
Since we are using Docker, all port mappings are handled inside the Dockerfile/ .yml file. But you are right, I should definitely change it cuz maybe some team members still run manually! Good catch!